easybuild/easyblocks/a/allinea.py

```diff
@@ -81,11 +84,30 @@ def install_step(self):
         # copy templates
         templ_path = os.path.join(self.installdir, 'templates')
         for templ in self.cfg['templates']:
+	    path = self.obtain_file(templ, extension='qtf')
+	    if path:
+		self.log.debug('Template file %s found' % path)
+	    else:
+		raise EasyBuildError('No template file named %s found', templ)
+
             try:
-                shutil.copy2(templ, templ_path)
+                shutil.copy(path, templ_path)
             except OSError, err:
                 raise EasyBuildError("Failed to copy template %s to %s: %s", templ, templ_path, err)
 
+        # copy system.config if requested
+	sysconf_path = os.path.join(self.installdir, 'system.config')
```

Comment:
🚨 tabs detected! 🚨

fixed in https://github.com/akesandgren/easybuild-easyblocks/pull/2